clap_mangen/tests/roff.rs

```diff
@@ -70,6 +70,13 @@ fn possible_values() {
     common::assert_matches_path("tests/snapshots/possible_values.bash.roff", cmd);
 }
 
+#[test]
+fn flag_without_value() {
```

Comment:
Did you verify the behavior of the test changed with clab_mangen changes? I see other tests using SetTrue but their snapshots didn't change, implying this PR didn't have an affect (see also my other comment).

Reply:
Hmm, I see i was being too naive with the solution.

Reply:
It's strange because the change definitely does work on the example provided in the bug report